React/wbooks/src/redux/rents/actions.js

```diff
@@ -15,15 +15,25 @@ export const fetchRents = bookId => async dispatch => {
 
 export const saveRents = localRents => dispatch => dispatch({ type: types.RENTS_SAVED, payload: localRents });
 
-export const saveWish = (bookId, user) => async dispatch => {
-  dispatch({ type: types.RENTS_WISHES_SAVING });
-  await authService.saveWish(bookId, user);
-  dispatch({ type: types.RENTS_WISHES_SAVED });
-  const wishes = await authService.fetchWishes(store.getState().session.user);
-  dispatch({ type: types.RENTS_WISHES_FETCHED, payload: wishes.data });
+export const saveWish = bookId => async dispatch => {
+  dispatch({ type: types.RENTS_WISHES_SAVE });
+  const saveResponse = await authService.saveWish(bookId, store.getState().session.userId);
```

Comment:
Don't import `store`, you can receive a parameter `getState` in each action. Use `export const saveWish = bookId => async (dispatch, getState) => {...`

Reply:
Also, call `getState` just one time in order to be more performant. Use destructuring assignment `const { user, userId } = getState.session;`